app/api/validations/user.py

```diff
@@ -71,3 +71,13 @@ def validate_resend_email_request_data(data):
         return {"message": "Your email is invalid."}
 
     return {}
+
+
+def validate_new_password(password):
```

Comment:
Can you add here an extra validation to see in the `current_password` and `new_password` are in data object? Because if a user doesn't send that when you call this function assuming that the `new_password` is being sent, you'll get an exception because you're trying to access the `new_password` in data when it does not exist. You can see the example on `validate_resend_email_request_data`.